api/client/trust.go

```diff
@@ -248,6 +252,8 @@ func notaryError(err error) error {
 		return fmt.Errorf("remote repository out-of-date: %v", err)
 	case trustmanager.ErrKeyNotFound:
 		return fmt.Errorf("signing keys not found: %v", err)
+	case *net.OpError:
```

Comment:
Yeah, just grepped and got the same. Somehow github's code search isn't returning it.